package.json

```diff
@@ -79,14 +79,15 @@
     "postcss-import": "^12.0.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.7.0",
+    "postcss-safe-parser": "^4.0.2",
```

Comment:
What is this one added for?

Reply:
Oops, this one snuck in when it shouldn't have!